### PR TITLE
Added readme.md

### PR DESCRIPTION
**Pull request by vkhv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# BEM react srub
```
Comment (awinogradov) [inline README.md]: stub

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Example
+
+Creating a block with element:
+
+blocks/header/header.js
+
+```
```
Comment (awinogradov) [inline README.md]: js after commas

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	content() {
+		return 'header';
+	}
+});
+```
+
+Using the block from root component:
+blocks/root/root.js
+
+```
```
Comment (awinogradov) [inline README.md]: here too

File: README.md (hunk 1 above)
Comment (vkhv) [inline README.md]: c4171eca38bef20a510f5402a3f04fed73f1216b

File: README.md (hunk 2 above)
Comment (vkhv) [inline README.md]: c4171eca38bef20a510f5402a3f04fed73f1216b

Comment (vkhv) [inline README.md]: c4171eca38bef20a510f5402a3f04fed73f1216b

File: README.md (hunk 3 above)
Comment (vkhv) [inline README.md]: c4171eca38bef20a510f5402a3f04fed73f1216b

**Comment by awinogradov** [comment]
@vkhv thanks!

**Pull request merged**